modules/core/src/test/java/org/apache/ignite/failure/FailureHandlingConfigurationTest.java

```diff
@@ -105,6 +110,60 @@
         cleanPersistenceDir();
     }
 
+    /**
+     * @throws Exception If failed
+     */
+    @Test
+    public void testGetorCreateLongCacheNameExceed235() throws Exception{
```

Comment:
I suggest renaming the test to smth. like `testCreateLongCacheName` and move it to `org.apache.ignite.internal.processors.cache.GridCacheConfigurationConsistencySelfTest` test class.